aws/sagemaker-rapids/sagemaker-rapids.ipynb

```diff
@@ -0,0 +1,373 @@
+{
```

Comment:
<p>We should probably link to the original blog post here somewhere.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>